code/src/Albumcontainer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Albumartistname } from './Albumartistname'
+import { Heart } from './Heart'
+import {Play} from './Play'
+import { Dots } from './Dots'
+
+
+export const Albumcontainer = props => {
+  console.log("We want album name:", props)
+    return (
+    <>
```

Comment:
You don't need the React fragment here since the `div` is wrapping all the content